run

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
In your opinion what is the advantage of having a run script over a docker file?

Reply:
This run script is only used when developing locally, so it is in charge of setting up and configuring the local development environment. A docker file typically won't have any environment information in it, so the environment configuration has to come from somewhere to facilitate local development.